immunisations/README.md

```diff
@@ -0,0 +1,149 @@
+| meta / profile                                           | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Immunization-1"`                 |
+| extension / url                                          | fixed value = `https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-VaccinationProcedure-1` |
+| extension / valueExtension / url                         | fixed value = `https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid`                       |
+| extension / valueExtension / valueCodeableConcept / text | `ObservationStatement / code / displayname`                                                                   |
+| extension / url[1]                                       | fixed value = `https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-DateRecorded-1`         |
+| extension / valueString                                  | `ObservationStatement / effectiveTime / highValue`                                                            |
+| identifier / system                                      | `ObservationStatement / code / displayname`                                                                   |
+| identifier / value                                       | `ObservationStatement / id`                                                                                   |
+| status                                                   | `ObservationStatement / statusCode`                                                                           |
+| notGiven                                                 | boolean value = `if passed through then set to true, otherwise it's false`                                    |
+| patient / reference                                      | reference to the mapped [patient](../patient/README.md)                                                       |
+| encounter / reference                                    | reference to the associated [encounter] ((../encounter/README.md))                                            |
+| date                                                     | `ObservationStatement / effectiveTime`                                                                        |
+| primarySource                                            | boolean value = `if passed through then set to true, otherwise it's false`                                    |
+| practitioner / actor / reference                         | `ObservationStatement / Participant / typeCode / agentRef / id / root`                                        |
+| note / text                                              | `ObservationStatement / pertiinentInformation / pertiantAnnotation / text`                                    |
+| note / text[1]                                           | `ObservationStatement / availabletime`                                                                        |

```

Comment:
I'd use `note\[0]` and `note\[1]` to denote the two notes.

Reply:
Hmm, these notes [look like they are working differently](https://github.com/NHSDigital/nia-patient-switching-standard-adaptor/blob/main/gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/ImmunizationMapper.java#L170-L178) to how the docs suggest here.